lnd/lnd.go

```diff
@@ -282,28 +282,23 @@ func (lnd *LND) GetChannelInfo(chanId uint64) (*lnrpc.ChannelEdge, error) {
 	})
 }
 
-func (lnd *LND) PayInvoice(invoice string, feeLimit uint, timeoutSeconds uint, chanId lightning.ChanId) (*lightning.PayInvoiceResponse, error) {
-
-	/*
-		var outgoungIds []uint64
-
-		if chanId != "" {
-			id, err := strconv.ParseInt(chanId, 10, 64)
-			outgoungIds = append(outgoungIds, uint64(id))
-
-			if err != nil {
-				return nil, err
-			}
+func (lnd *LND) PayInvoice(invoice string, feeLimit uint, timeoutSeconds uint, chanIds []lightning.ChanId) (*lightning.PayInvoiceResponse, error) {
+	var outgoungIds []uint64
+	for _, chanId := range chanIds {
+		if chanId != 0 {
+			outgoungIds = append(outgoungIds, uint64(chanId))
 		}
-	*/
+	}
+
+	fmt.Println(outgoungIds)
 
 	client, err := lnd.router.SendPaymentV2(lnd.ctx, &routerrpc.SendPaymentRequest{
 		PaymentRequest:    invoice,
 		TimeoutSeconds:    int32(timeoutSeconds),
 		FeeLimitSat:       int64(feeLimit),
 		NoInflightUpdates: true,
 		// TODO: this is not working right now
```

Comment:
This you manage to reliable reproduce the flakiness?

Reply:
In regtest:
```bash
clnId=$(lightning-cli-sim 1 getinfo | jq -r  .id)
chanId=$(lncli-sim 1 listchannels --peer $clnId | jq -r  ".channels.[].chan_id")
./boltzcli createreverseswap --chan-id $chanId  100000
```

Reply:
its a regtest issue, works in testnet